components/python/pytest-perf/Makefile

```diff
@@ -0,0 +1,32 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# This file was automatically generated using the following command:
+#   $WS_TOOLS/python-integrate-project pytest-perf
+#
+
+BUILD_STYLE = setup.py
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME =		pytest-perf
+COMPONENT_VERSION =		0.12.0
+COMPONENT_SUMMARY =		pytest-perf - pytest-perf
```

Comment:
This summary is not sufficient. What about
COMPONENT_SUMMARY= Run performance tests against the mainline code
from the pypi project page?

Reply:
You are right.  Fixed.